site/pages/api/change-password.test.ts

```diff
@@ -0,0 +1,99 @@
+import { describe, it, expect, afterEach, vi } from "vitest";
+import changePassword from "./change-password.api";
+import { CHANGE_PASSWORD_PAGE_PATH, COOKIES_CHANGE_PASSWORD_ERRORS } from "../../constants";
+import { ErrorInfo } from "../../interfaces";
+import { getMockRequestAndResponse } from "../../testData/mockData";
+import { destroyCookieOnResponseObject, setCookieOnResponseObject } from "../../utils/apiUtils";
+
+describe("changePassword", () => {
+    const writeHeadMock = vi.fn();
+    vi.mock("../../utils/apiUtils", async () => ({
+        ...(await vi.importActual<object>("../../utils/apiUtils")),
+        setCookieOnResponseObject: vi.fn(),
+        destroyCookieOnResponseObject: vi.fn(),
+    }));
+
+    afterEach(() => {
+        vi.resetAllMocks();
+    });
+
+    it("should redirect to login page when incorrect inputs are passed", () => {
```

Comment:
I think this test name should be 'when no inputs are passed' rather than 'incorrect' ones.

Reply:
done